### PR TITLE
Fxing pr test to use code in pr

### LINKED ISSUE #928: Jenkins komodo build test not using built lib file
For PRs the jenkins komodo test script is not picking up the library files of the PR version of the library. This means that new code is not tested. Should be changed to be picking up the new version of libres and the dependencies from komodo.

### PR DESCRIPTION
**Pull request by ManInFez**
resolves #928

### CODE REVIEW

**Comment by ManInFez** [comment]
Test this please!

**Comment by ManInFez** [comment]
Test this please!

**Comment by ManInFez** [comment]
Test this please!

**Comment by ManInFez** [comment]
Test this please!

**Comment by ManInFez** [comment]
Test this please!

**Comment by ManInFez** [comment]
Test this please!

File: ci/jenkins/testkomodo.sh [hunk 1]
```diff
@@ -54,6 +54,12 @@ if [[ -z "${sha1// }" ]]; then
     mkdir tmptest
     cp -r python/tests tmptest/tests
     pushd tmptest
+else
+    export ERT_SITE_CONFIG=$(pwd)/share/ert/site-config
+    PYV=`python -c "import sys;t='{v[0]}.{v[1]}'.format(v=list(sys.version_info[:2]));sys.stdout.write(t)";`
```
Comment (markusdregi) [inline ci/jenkins/testkomodo.sh]: How about `python -c "import sys;print('{}.{}'.format(*sys.version_info[:2]))"` instead?

**Review by markusdregi — APPROVED** [review]
Had a simplification suggestion for fetching the Python version. Besides that this looks good to me 👍

**Comment by markusdregi** [comment]
Btw, you also have a typo in your commit message...

**Comment by ManInFez** [comment]
Test this please!

**Comment by ManInFez** [comment]
Test this please!

**Pull request merged**